mrtt-ui/src/views/Sites.js

```diff
@@ -24,19 +24,32 @@ function Sites() {
       .catch(() => toast.error(language.error.apiLoad))
   }, [])
 
-  const sitesList = sites.map(({ site_name, id }) => (
-    <LinkCard key={id} to={`/site/${id}`}>
-      {site_name}
-    </LinkCard>
-  ))
+  const sitesList = sites
+    .sort((siteA, siteB) => {
```

Comment:
if we werent moving quick, Id spend time naming this function better. Might do next time I need to use sort.